src/collections/members/tolulope-ola-david/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Tolulope Ola-David
+position: Contributor
+image_path: ./tolulope-ola-david.png
+github: TolulopeMO
+twitter: tolulope_ola
+linkedin: tolulopeoladavid
+location: Lagos, Nigeria
+bio: Tolulope Ola-David is a highly skilled technical and content writer with over seven years of experience in the field. He has a proven track record of creating compelling and engaging written content for a variety of audiences and purposes and uses his technical expertise to create clear, concise, and accurate documentation. Tolulope is proficient at conducting research, organizing information, and adapting his writing style to fit the needs of the project. He has excellent communication and interpersonal skills, as well as a passion for staying up-to-date on industry trends and best practices.
+badges: 
+    - community
```

Comment:
@Anita-ihuman @DivineOrji here's an excellent opportunity to apply the `writing` badge.

@chinu-anand, we'll need to swap `community` with `writing`.

Reply:
@chinu-anand excellent. In the meantime, change this PR, so that it can be merged as well.

Reply:
@leecalcote I have changed the badge from `community` to `writer`.